tools/add_header

```diff
@@ -65,6 +65,7 @@ author_map = {
     ("maatthias", "<maatthias@gmail.com>"): ("Matt Snyder", "<msnyder@bnl.gov>"),
     ("maatthias", "<msnyder@bnl.gov>"): ("Matt Snyder", "<msnyder@bnl.gov>"),
     ("Mario Lassnig", "<mlassnig@users.noreply.github.com>"): ("Mario Lassnig", "<mario.lassnig@cern.ch>"),
+    ("", "jbogado@linti.unlp.edu.ar"): ("jbogado@linti.unlp.edu.ar"),
```

Comment:
second entry is not a tuple and is missing the name :)

Reply:
Hmm not sure why this is part of the PR, but it shouldn't be :-D

Reply:
Fixed 👍